pandas/core/frame.py

```diff
@@ -127,7 +127,8 @@
     * left: use only keys from left frame (SQL: left outer join)
     * right: use only keys from right frame (SQL: right outer join)
     * outer: use union of keys from both frames (SQL: full outer join)
-    * inner: use intersection of keys from both frames (SQL: inner join)
+    * inner: use intersection of keys from both frames (SQL: inner join),
+        preserving the order of the left keys
```

Comment:
small rst nitpick (it's very whitespace sensitive ..): the 'preserving ..' should be aligned with the 'inner ..' on the line above